plugins/repository-gcs/src/internalClusterTest/java/org/elasticsearch/repositories/gcs/GoogleCloudStorageBlobStoreRepositoryTests.java

```diff
@@ -310,15 +310,15 @@ protected boolean canFailRequest(final HttpExchange exchange) {
         @Override
         public void maybeTrack(final String request, Headers requestHeaders) {
             if (Regex.simpleMatch("GET /storage/v1/b/*/o/*", request)) {
-                trackRequest("storage.objects.get");
+                trackRequest("GetObject");
             } else if (Regex.simpleMatch("GET /storage/v1/b/*/o*", request)) {
-                trackRequest("storage.objects.list");
+                trackRequest("ListObjects");
             } else if (Regex.simpleMatch("GET /download/storage/v1/b/*", request)) {
-                trackRequest("storage.objects.get");
+                trackRequest("GetObject");
             } else if (Regex.simpleMatch("PUT /upload/storage/v1/b/*", request) && isLastPart(requestHeaders)) {
```

Comment:
Should this be changed to match `PUT /upload/storage/v1/b/*uploadType=resumable*`? AFAICS, resumable sessions always identify sub-requests that way? Simpler than that `isLastPart` stuff

Reply:
Resumable uploads are billed as a single operation, in this case we're only tracking the last operation.

Reply:
ok, let's add a comment to that effect. Still allows us to match on `*uploadType=resumable*` here?